src/markdown-pages/automate-workflows/error-inbox/spin-up-geek-movie.mdx

```diff
@@ -0,0 +1,95 @@
+---
+path: '/automate-workflows/error-inbox/spin-up-geek-movie'
+title: 'Spin up Geek movie architecture'
+template: 'GuideTemplate'
+description: 'Set up your your environment to deploy geek movie shop.'
+duration: 20
+procIdx: 1
+---
+
+<Callout variant="course" title="lab">
+
+This procedure is part of a lab that teaches you how to track full stack errors using Error Inbox. If you haven't already, check out the [lab introduction](/automate-workflows/error-inbox).
+
+</Callout>
+
+Geek's Movie Shop (former Stan's Robot Shop) is a web application that consists of several interconnected microservices like cart, catalogue, payment, user, and more. These services are instrumented with New Relic for performance monitoring and full visibility across all layers. The application also has a loader service that generates traffic In this lab, we use New Relic's Error Inbox for full stack error tracking.
+
+## Set up your environment
+
+Before you begin with the lab, set up your development environment. 
+<Steps>
+<Step>
+Clone the lab repository:
+
+```bash
+git clone https://github.com/newrelic-experimental/error-inbox-lab-materical.git
+```
+
+</Step>
+<Step>
+Open the cloned repo in your IDE and create a .env file from env.template and set these variables:
+
+```
+DOCKERHUB_USER=username
+TAG=latest
+CLUSTER_NAME=local
+PUBLIC_URL=http://docker-for-desktop:8888
+
+NEW_RELIC_LICENSE_KEY=<YOUR_LISCENCE_KEY>
+```
+
+Here, NEW_RELIC_LICENSE_KEY is your liscence key for New Relic account. Your application uses this key to write telemetry data to New Relic. [You can find your New Relic license key in your account settings](https://docs.newrelic.com/docs/apis/intro-apis/new-relic-api-keys/).
+
+</Step>
+<Step>
+
+Navigate to the root directory of your application and export these variables:
+
+```bash
+./env.sh
```

Comment:
.env file is where reader declares all the variables.  env.sh file is already provided. They just have to run it to export all the variable so they don' have to manually export all the variables.

Reply:
Why not just use a single file?

Reply:
The reason I'm asking is because this three file operation feels overly complicated. It seems like you should just have a .env file in the repo and ask them to clone the repo and add their license key and then run:

```
source .env
```

---

Either way, this didn't work for me:

```
$ ./env.sh
$ echo $DOCKERHUB_USER

$ source .env
$ echo $DOCKERHUB_USER
username
```